graylog2-web-interface/src/components/grok-patterns/GrokPatternQueryHelper.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow strict
+import React from 'react';
```

Comment:
Does this need to be 
```suggestion
import * as React from 'react';
```
for Flow?

Reply:
Flow wasn't crying. But I still think it is a good idea :)

Reply:
I think the more common way is to `import React from 'react';` as the default export is all the named exports combined. It's pretty much a convention at this point.